src/pudl/package_data/eia923/column_maps/fgd_operation_maintenance.csv

```diff
@@ -0,0 +1,19 @@
+fgd_land_acquisition_costs_thousand_dollars,land_acquisition,,,,,,,,,,,,,,
+fgd_waste_disposal_costs_thousand_dollars,waste_disposal,waste_disposal,waste_disposal,waste_disposal,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars
+fgd_maintenance_costs_thousand_dollars,,,,,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars
+fgd_maintenance_material_other_costs_thousand_dollars,maintenance_material_other,maintenance_material_other,maintenance_material_other,maintenance_material_other,,,,,,,,,,,
+total_thousand_dollars,total_o_m,total_o_m,total_o_m,total_o_and_m,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars
+fgd_control_flag,no_fgd_control,no_fgd_control,no_fgd_control,no_fgd_control,,,,,,,,,,,
+fgd_operational_status,fgd_unit_status,fgd_unit_status,fgd_unit_status,fgd_unit_status,,,,,,,,,,,
+fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,,,,,,,,,,,
+fgd_sorbent_consumption_1000_tons,quantity_of_sorbent,quantity_of_sorbent,quantity_of_sorbent,quantity_of_sorbent,,,,,,,,,,,
+fgd_electricity_consumption_mwh,electric_energy_consumption,electric_energy_consumption,electric_energy_consumption,electric_energy_consumption,,,,,,,,,,,
+so2_removal_efficiency_annual,efficiency_at_annual_operating_factor,efficiency_at_annual_operating_factor,efficiency_at_annual_operating_factor,efficiency_at_annual_operating_factor,,,,,,,,,,,
```

Comment:
The names in EIA have changed over time, but generally the SO2 columns refer to the particulate (e.g., how efficient the given control system is at removing it), while the FGD prefix refers to the actual FGD technology itself. Since there are other ways of lowering SO2 emissions, I'd prefer to keep the distinction clear. This is also replicating what's already in the `emissions_control` table.